### ISSUE TITLE
Please update "sys" to "utils"

### ISSUE THREAD

**Issue by gubi** [nisaacson/pdf-extract#16]
When execute a node script from my terminal this message appear:

``` text
(node:21492) DeprecationWarning: sys is deprecated. Use util instead.
```

As you can [read here](https://github.com/nodejs/node-v0.x-archive/issues/3577#issuecomment-6635361), there's no more chances in the future to use sys.<br />
Can you update the repo?

Thanks

**Comment by rightisleft** [comment]
+ 1

**Comment by nisaacson** [comment]
merged via #22, thanks @mchapman
